### PR TITLE
Drop gulp-rimraf in favour of del. Closes #276

### LINKED ISSUE #276: gulp-rimraf is deprecated
Deploying to Heroku tells me this:

```
npm WARN deprecated gulp-rimraf@0.1.1: Use npmjs.org/del instead, see https://github.com/gulpjs/gulp/blob/master/docs/recipes/delete-files-folder.md
```

### PR DESCRIPTION
**Pull request by jbrooksuk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
     "bower": "1.3.12",
+    "del": "1.1.1",
```
Comment (GrahamCampbell) [inline package.json]: shouldn't lock dependencies like this

Comment (GrahamCampbell) [inline package.json]: is `~1.1.1` if the project is non-semver, or `^1.1.1` if it is semver.

File: package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
     "bower": "1.3.12",
```
Comment (GrahamCampbell) [inline package.json]: same here

File: package.json (hunk 1 above)
Comment (jbrooksuk) [inline package.json]: How come? Want me to use `^1.1.1`

File: package.json (hunk 2 above)
Comment (jbrooksuk) [inline package.json]: Ok, I'll fix them up.

File: package.json (hunk 1 above)
Comment (GrahamCampbell) [inline package.json]: It makes no sense to lock it at that version. What's wrong with 1.1.2 when it's released?

Comment (jbrooksuk) [inline package.json]: I've set it to `~1.1.1`.

**Pull request merged**